### PR TITLE
DM-13997: {{Add template doc/ infrastructure}}

### PR DESCRIPTION
**Pull request by ebellm**

### CODE REVIEW

File: doc/manifest.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Documentation manifest.
+
+# Name of this package
+# Also the name of the package documentation subdirectory.
+package: "ip_diffim"
```
Comment (jonathansick) [inline doc/manifest.yaml]: `ip_isr`

File: doc/manifest.yaml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+# List of names of Python modules in this package.
+# For each module there is a corresponding module doc subdirectory.
+modules:
+  - "lsst.ip.diffim"
```
Comment (jonathansick) [inline doc/manifest.yaml]: `lsst.ip.isr`

File: doc/manifest.yaml [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+# Name of the static content directories (subdirectories of `_static`).
+# Static content directories are usually named after the package.
+statics:
+  - "_static/ip_diffim"
```
Comment (jonathansick) [inline doc/manifest.yaml]: `ip_isr`

**Comment by ebellm** [comment]
Fixed, sorry about that.  Can you give it the green light?

**Pull request merged**